tests/loader/test_hilti_loader.py

```diff
@@ -4,14 +4,75 @@
 """
 import unittest
 from pathlib import Path
+from typing import Iterable, List, Tuple
 
 import numpy as np
+from gtsam import (
+    BetweenFactorPose3,
+    InitializePose3,
+    LevenbergMarquardtOptimizer,
+    NonlinearFactorGraph,
+    Point3,
+    Pose3,
+    PriorFactorPose3,
+    Rot3,
+    Values,
+    noiseModel,
+)
 
-import gtsfm.utils.geometry_comparisons as comp_utils
+from gtsfm.common.constraint import Constraint
 from gtsfm.loader.hilti_loader import HiltiLoader
 
 DATA_ROOT_PATH = Path(__file__).resolve().parent.parent / "data"
 TEST_DATASET_DIR_PATH = DATA_ROOT_PATH / "hilti_exp4_small"
+EXP07_PATH = DATA_ROOT_PATH / "exp07"
+
+
+def pose_of_vector(pose_vector):
+    """Convert from vector to Pose3"""
+    pose_rotation = Rot3(pose_vector[-1], pose_vector[3], pose_vector[4], pose_vector[5])
+    pose_translation = Point3(pose_vector[:3])
+    return Pose3(pose_rotation, pose_translation)
+
+
+def create_initial_estimate(poses: List[Pose3]):
+    """Create initial estimate"""
+    initial_estimate = Values()
+    for i, pose_vector in enumerate(poses):
+        initial_estimate.insert(i, pose_vector)
+    return initial_estimate
+
+
+def read_fastlio_result(poses_txt_path: str) -> Tuple[List[Pose3], np.ndarray]:
+    """
+    Read FastLIO result
```

Comment:
@ayushbaid can we update this docstring to the style guide when you get a chance?